samples/README.md

```diff
@@ -40,8 +40,18 @@ View the [source code](https://github.com/googleapis/nodejs-firestore/blob/maste
 
 __Usage:__
 
+<<<<<<< HEAD
```

Comment:
@bcoe looks like this guy snuck in here!

Reply:
Good catch! fixing.